Options.py

```diff
@@ -739,8 +739,16 @@ def __str__(self) -> str:
         return str(self.value)
 
     @staticmethod
-    def triangular(lower: int, end: int, tri: typing.Optional[int] = None) -> int:
-        return int(round(random.triangular(lower, end, tri), 0))
+    def triangular(lower: int, end: int, tri: float = 0.5) -> int:
+        """
+        Integer triangular distribution for `lower` inclusive to `end` inclusive.
+
+        Expects `lower <= end` and `0.0 <= tri <= 1.0`. The result of other inputs is undefined.
+        """
+        # Use the continuous range [lower, end + 1) to produce an integer result in [lower, end].
+        # random.triangular is actually [a, b] and not [a, b), so there is a very small chance of getting exactly b even
+        # when a != b, so ensure the result is never more than `end`.
+        return min(end, math.floor(random.triangular(0.0, 1.0, tri) * (end - lower + 1) + lower))
```

Comment:
"lower" was getting confusing for me, you could use "start" instead, maybe? I think "mode" is also more descriptive than "tri" for this distribution.
```suggestion
    def triangular(start: int, end: int, mode: float = 0.5) -> int:
        """
        Integer triangular distribution from `start` inclusive to `end` inclusive.

        Expects `start <= end` and `0.0 <= mode <= 1.0`. The result of other inputs is undefined.
        """
        # Use the continuous range [start, end + 1) to produce an integer result in [start, end].
        # random.triangular is actually [a, b] and not [a, b), so there is a very small chance of getting exactly b even
        # when a != b, so ensure the result is never more than `end`.
        return min(end, math.floor(random.triangular(0.0, 1.0, mode) * (end - start + 1) + start))
```

Reply:
Would be on board with this personally, however since tri is a kwarg, we'd have to make sure it's not passed as `tri=0.8` anywhere

Reply:
Why? That's an intended mechanic

Reply:
I'm saying if we change the name of the arg, then we have to check whether any worlds are currently passing the old name of the arg (in some non-unit test covered code)

Reply:
Oh 🤦 doing a quick search, I don't see any instances of that